ds/cmd/config.sh

```diff
@@ -29,4 +29,6 @@ cmd_config() {
 
     # drush may create some files with wrong permissions, fix them
     ds inject fix-file-permissions.sh
+
+    ds inject install/redis-config.sh
```

Comment:
Place this line after: `ds inject install/drupal-config.sh`

Reply:
I have tested which line should this script is executed,
it must be run after drupal, drush, and apache is installed and running